packages/common/src/GenericContract.ts

```diff
@@ -2,7 +2,7 @@ import { Buffer } from 'buffer/';
 import { stringify } from 'json-bigint';
```

Comment:
I could be overlooking something, but I don't see how `Contract` in this file is public anymore. The client generator relies on it being public

Reply:
Must have forgotten to export it. Will add this.